documentation/plugin-dev-guide.html

```diff
@@ -113,12 +122,12 @@ <h2>Structure of a Plugin</h2>
     <span class="comment">&lt;!-- Plugin meta-data --&gt;</span>
     &lt;name&gt;Example Plugin&lt;/name&gt;
     &lt;description&gt;This is an example plugin.&lt;/description&gt;
-    &lt;author&gt;Jive Software&lt;/author&gt;
+    &lt;author&gt;Your Name&lt;/author&gt;
 
-    &lt;version&gt;1.0&lt;/version&gt;
-    &lt;date&gt;07/21/2006&lt;/date&gt;
+    &lt;version&gt;1.0.0&lt;/version&gt;
+    &lt;date&gt;2006-07-21&lt;/date&gt;
```

Comment:
Should we bump this to something more recent?
This plugin would be old enough to drink in many places :)